GUI Enhancements

gamecoder95 opened the pull request:
- Added response label for bidding, as well as a status label to indicate if the client is winning or not.
- Formatted text on GUI so that it breaks it into multiple lines for long messages.
- When a list item is removed, it clears the corresponding info panel if it was the selected item.

gamecoder95 committed:
Added response label for bidding

Bidding now has a proper label for the responses.

gamecoder95 committed:
Added error message for bidding

When trying to bid for an unselected item, an error message appears
instead.

samratdebroy reviewed (approved):
Everything looks good to me!

gamecoder95 committed:
Added response text formatting

Formatted the response text by programmatically inserting new line
escape characters every 8 words.

gamecoder95 committed:
GUI formatting fixes

Fixed issues with formatting, when a list item is removed and it was the
selected one, it clears the concerning info panel.

gamecoder95 committed:
Added winning label

When the client has the highest bid, it updates the "Winning" field to
"Yes"; "No" otherwise.

gamecoder95 committed:
Improved clearing upon removal of list item

Removed checking for selected indices for "conditional clearing"
methods.
